pkg/serverless/appsec/httpsec/http_test.go

```diff
@@ -19,58 +17,78 @@ import (
 	"github.com/DataDog/datadog-agent/pkg/serverless/invocationlifecycle"
 	"github.com/DataDog/datadog-agent/pkg/trace/api"
 	"github.com/DataDog/datadog-agent/pkg/trace/sampler"
+	"github.com/stretchr/testify/assert"
 	"github.com/stretchr/testify/require"
 )
 
 func TestLifecycleSubProcessor(t *testing.T) {
-	t.Setenv("DD_SERVERLESS_APPSEC_ENABLED", "true")
-	asm, err := appsec.New()
-	if err != nil {
-		t.Skipf("appsec disabled: %v", err)
-	}
+	test := func(t *testing.T, appsecEnabled bool) {
+		t.Setenv("DD_SERVERLESS_APPSEC_ENABLED", strconv.FormatBool(appsecEnabled))
+		asm, _, err := appsec.New()
+		require.NoError(t, err)
```

Comment:
```suggestion
		if err != nil {
			t.Skipf("appsec disabled: %v", err)
		}
```

Reply:
FYI, this is how we have been skipping our tests on our unsupported environments so far.